### DISCUSSION TITLE
Issue with compiling CacheFunction.js

### DISCUSSION THREAD

**Discussion by DanMDA1** [demmings/cachefinance#9] [Q&A]
Hi,

I'm getting the following error when compiling the CacheFunction.js code. Did I miss something?

```
TypeError: Cannot read properties of undefined (reading 'toUpperCase')
getFinanceData	@ GoogleFinCache.gs:56
CACHEFINANCE	@ GoogleFinCache.gs:36
```

**Comment by demmings** [comment]
Dan,
Can you copy in the exact CACHEFINANCE() syntax you used so I can try in my test sheet:

https://docs.google.com/spreadsheets/d/1zk7GIJ7ys4hBhiZwrLemqIGf-lH7eiLcJWKw6IkdAY4/edit?usp=sharing

The test sheet is probably not exactly what I have posted in Github since I don't put stuff not fully tested. (so github should be stable while I test on this sheet).  You can also try on that sheet as well.  Hopefully this link allows update access.

Chris.
